app/services/proxy_status/base.rb

```diff
@@ -25,7 +25,7 @@ def self.humanized_name
     def api
       @api = "ProxyAPI::#{self.class.humanized_name}".classify.constantize.new(:url => proxy.url)
     rescue NameError => e
-      raise Foreman::WrappedException.new("ProxyAPI::#{self.class.humanized_name}", e, N_('Unable to initialize ProxyAPI class'))
```

Comment:
This one fell through the cracks, I think it was changed when I commented on it then reverted and I didn't notice. http://projects.theforeman.org/projects/foreman/wiki/Translating had the order of WrappedException parameters reversed so I changed that